aws/internal/keyvaluetags/s3_tags.go

```diff
@@ -87,6 +87,10 @@ func S3ObjectListTags(conn *s3.S3, bucket, key string) (KeyValueTags, error) {
 
 	output, err := conn.GetObjectTagging(input)
 
+	if awsbase.IsAWSErr(err, "NoSuchTagSet", "") {
```

Comment:
Prevents errors:

```console
$ make testacc TEST=./aws/ TESTARGS='-run=TestAccDataSourceAWSS3BucketObject_SingleSlashAsKey'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws/ -v -count 1 -parallel 20 -run=TestAccDataSourceAWSS3BucketObject_SingleSlashAsKey -timeout 120m
=== RUN   TestAccDataSourceAWSS3BucketObject_SingleSlashAsKey
=== PAUSE TestAccDataSourceAWSS3BucketObject_SingleSlashAsKey
=== CONT  TestAccDataSourceAWSS3BucketObject_SingleSlashAsKey
--- FAIL: TestAccDataSourceAWSS3BucketObject_SingleSlashAsKey (24.58s)
    testing.go:684: Step 0 error: errors during apply:
        
        Error: error listing tags for S3 Bucket (tf-acc-test-2642544354113174819) Object (/): NoSuchTagSet: The TagSet does not exist
        	status code: 404, request id: 3ADF2BE8D6DAD57E, host id: zIz72pHcPivqgl0t79J/9kF02E5N7ibZy3Rof+ggH2+Y3sffOQsjRQVkP5B/PinBmLJnI1nfvCk=
        
          on /tmp/tf-test734233549/main.tf line 6:
          (source code not available)
        
        
    testing.go:745: Error destroying resource! WARNING: Dangling resources
        may exist. The full state and error is shown below.
        
        Error: errors during refresh: error listing tags for S3 Bucket (tf-acc-test-2642544354113174819) Object (/): NoSuchTagSet: The TagSet does not exist
        	status code: 404, request id: 9746D2A0387F2F59, host id: tCpkS+zPf8miGLqN8BaDPGbPBs03PZoC3iJHWvhd3SeN7k/Y1BD1AKJmujxgfBqGshau1OP9Pu4=
        
        State: <nil>
FAIL
FAIL	github.com/terraform-providers/terraform-provider-aws/aws	24.654s
FAIL
GNUmakefile:26: recipe for target 'testacc' failed
make: *** [testacc] Error 1
```

Reply:
After recent merge, ready for rebase. 👍 

```
# github.com/terraform-providers/terraform-provider-aws/aws/internal/keyvaluetags
aws/internal/keyvaluetags/s3_tags.go:91:5: undefined: awsbase
```

```suggestion
	if tfawserr.ErrCodeEquals(err, tfs3.NoSuchTagSet) {
```

Reply:
Will do (plus change the error code constants to include `ErrCode` prefix).